participants/scipy_000/pr_tutorial/buggy_function.py

```diff
@@ -20,7 +20,7 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
     if math.floor(decimals) != decimals:
         raise OSError('decimals should be an integer!')
 
-    hours_num = angle_in_degrees*24/180
+    hours_num = angle_in_degrees*24/360
```

Comment:
What test did you do to demonstrate this fixed the function? What do you think about adding a unit test to tests/?

Reply:
Good idea! I added a basic unit test based off the other example test function.